cni-metrics-helper/README.md

```diff
@@ -0,0 +1,75 @@
+# cni-metrics-helper
+
+The `cni-metrics-helper` is a tool that can be used to scrape the aggregated ENIs and IPs information at the cluster 
+level and publish the metrics to CloudWatch. In able to publish metrics with useful names, the following two permissions
```

Comment:
"The following two IAM permissions are required by the worker nodes to publish metrics and provide useful metric names:"

?